lib/chef/resource/windows_security_policy.rb

```diff
@@ -43,6 +43,35 @@ class WindowsSecurityPolicy < Chef::Resource
       description "Use the **windows_security_policy** resource to set a security policy on the Microsoft Windows platform."
       introduced "16.0"
 
+      examples <<~DOC
+      **Set Administrator Account to Enabled**:
+
+      ```ruby
+      windows_security_policy 'EnableAdminAccount' do
+        secvalue       '1'
```

Comment:
this extra spacing might cause issues with cookstyle. You'll want to make sure that this passes so that a user can copy / paste it into a recipe without getting a warning.